cryptoConsole/cryptowatch.py

```diff
@@ -21,84 +21,105 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-import argparse
 
-from time import sleep
-from os.path import exists
-from sys import platform
+import argparse
+from os.path import exists, expanduser
 from re import sub
-from os.path import expanduser
-
+from sys import platform
+from time import sleep
 
+import cryptoUtils.cwconfig as cfg
 from cryptoUtils.cryptoUtils import clear
 from cryptoUtils.cryptoUtils import getCryptoTable
-import cryptoUtils.cwconfig as cfg
+
+HEADER = r'''
```

Comment:
Can you explain what the r does here? The code runs fine with the r but it throws off the syntax highlighting for the rest of the file in my editor. Without the r the header doesn't print out correctly is there another way to replicate this same functionality (hopefully that other way will not mess with my syntax highlighting)

Reply:
Oh, it's just using a raw string so you don't need the backslash escape characters.

> [Both string and bytes literals may optionally be prefixed with a letter 'r' or 'R'; such strings are called raw strings and treat backslashes as literal characters. As a result, in string literals, '\U' and '\u' escapes in raw strings are not treated specially. Given that Python 2.x’s raw unicode literals behave differently than Python 3.x’s the 'ur' syntax is not supported.](https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals)

Reply:
I just like to try and get all the PEP8 warning bars to disappear in PyCharm so I turned to that because the backslash characters were causing some lines to go over 79 chars.

It's your project though and I will change it back if you want.

Reply:
I honestly don't mind the change and do like to stick to the standards where possible, but it makes the rest of the file come up as one long string in my editor (in terms of syntax highlighting) is there another way that we could do this that will not break the standard but does not use a raw string?

Reply:
Thanks for posting the explanation on raw strings though, very interesting

Reply:
Could I ask what editor you are using? Or maybe take a screenshot so I could see? I'm just not exactly sure what you mean.

Reply:
I'll just change it back.

Reply:
Im using atom, I opened it up and neovim and there was no syntax highlighting issues. 
![bad](https://user-images.githubusercontent.com/16146094/34316834-e18371be-e76c-11e7-8b17-75e791cbd5a8.png)
![good](https://user-images.githubusercontent.com/16146094/34316835-e5025a58-e76c-11e7-85b5-5f6e49c7e98c.png)

Reply:
Ah ok, I see what you mean in that first pic. I should have waited for your response.

I'm not sure which way you prefer, just let me know what works better for you.

Reply:
I'd rather it without the raw string purely because atom is being weird, thanks for reverting the change

Reply:
No problem! Sorry for causing the issue!